modules/sb-install-service-catalog.adoc

```diff
@@ -2,55 +2,48 @@
 //
 // * applications/service_brokers/installing-service-catalog.adoc
 
-[id='sb-enable-service-catalog-{context}']
-= Enabling service catalog
+[id='sb-install-service-catalog-{context}']
+= Installing service catalog
 
-Service catalog is included by default in {product-title}, but you must you must
-enable it using the procedure below.
+Service catalog is not installed by default in {product-title} 4. You can install
+it by using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
+.. Use the following command to edit the service catalog API server resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogAPIServer
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogapiservers
 ----
-.. Use the following command to create the service catalog API server.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-api-server.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog API server is now installed into the `kube-service-catalog`
-namespace.
+This causes the Operator to install the service catalog API server
```

Comment:
I'd probably say "The Operator installs the service catalog API server in the \`openshift-service-catalog-apiserver\` namespace."

> Avoid sentences in which a pronoun can refer to more than one noun or in which a pronoun refers to an unstated noun.